.github/workflows/build-and-release.yml

```
@@ -0,0 +1,59 @@
+name: Build and Release BF Client
+
+# We use this action to build the client when we make changes so it is easily downloadable by chef on buildfarm servers
+
+on:
+  push:
+    branches:
+      - lucid
+  workflow_dispatch:
+
+permissions:
+  contents: write
+
+jobs:
+  build-and-release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Bazel
+        uses: bazel-contrib/setup-bazel@0.8.5
+        with:
+          bazelisk-cache: true
+          disk-cache: ${{ github.workflow }}
+          repository-cache: true
+
+      - name: Build ARM client
+        run: bazel build //:client_arm
```

We should get rid of the extra target just for arm and use this instead.
```suggestion
        run: bazel build //:client --platforms=@io_bazel_rules_go//go/toolchain:linux_arm64
```

---

Ok. we can revert Dalyn's commit adding them in a separate PR, but I'll switch to using your suggestion.